README.md

```diff
@@ -39,11 +39,11 @@ npm start
 
 # In another shell, run grafton.
 grafton test --product=bonnets --plan=small --region=aws::us-east-1 \
-    --client-id=$CLIENT_ID \
-    --client-secret=$CLIENT_SECRET \
-    --connector-port=$CONNECTOR_PORT \
+    --client-id=21jtaatqj8y5t0kctb2ejr6jev5w8 \
```

Comment:
The `client-id` and `client-secret` should still be specified by environment variable as they are consumed by the running application in index.js.